packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [29.0.1]
+
+### Fixed
+
+- transaction-controller – approveTransaction was throwing away raw signed transaction ([#4255](https://github.com/MetaMask/core/pull/4255))
```

Comment:
Nit: The change entry you included in the PR description here seems better suited. The "transaction-controller - " part here especially seems out-of-place (this whole changelog is transaction controller specific already).

I've added my own attempt at briefly summarizing this, then included your change entry as a sub bullet. Let me know what you think!

```suggestion
- Prevent pending transaction from being inaccurately identified as failed ([#4255](https://github.com/MetaMask/core/pull/4255))
  - When resubmitting a pending transaction, if an 'already seen' error is caught (which indicates that the transaction is still pending in the mempool), we now ignore the error instead of failing the transaction
```